### ISSUE TITLE
"Unavailable" messages

### ISSUE THREAD

**Issue by elyobelyob** [timmo001/ovoenergy#8]
Hi,

Did an upgrade to 0.118.4 yesterday and today see "unavailable", but no logs in home-assitant.log

Status shows as red and "restored" under entities.

Just wondering if anyone else has seen this or a known issue.

Thanks.

<img width="506" alt="Screenshot 2020-12-01 at 16 18 08" src="https://user-images.githubusercontent.com/298022/100766743-e7db2b00-33f0-11eb-9dc2-53baf5196dd8.png">

**Comment by elyobelyob** [comment]
Could this be due to the reauth issue with the fix on its way? https://github.com/home-assistant/core/pull/38882

**Comment by elyobelyob** [comment]
Also, I have removed and reimplemented the OVO integration with no luck ...

**Comment by elyobelyob** [comment]
I've also tested with postman to access my account, and the basic auth works ... thanks

**Comment by elyobelyob** [comment]
OK, I discovered the issue using postman.

I'm not yet on a smart meter, so have been regularly inputting my manual reading. However, with the change of the month to the 1st December, the null results aren't handled. So, rather than get a zero response, I was getting unavailable. I presume that even with a smart meter, everyone will show unavailable until the first reading is sent. So, would suggest handling it with a zero.

Here's what I think was received (I didn't exactly note the 1st Dec reading and sent a reading, by sending a month before I joined, I get this ... which I think is the same).

{
    "electricity": null,
    "gas": null
}
